hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -17,7 +17,11 @@
 package com.hazelcast.instance;
 
 import com.hazelcast.config.Config;
+import com.hazelcast.core.IMap;
 import com.hazelcast.internal.monitors.HealthMonitorLevel;
+import com.hazelcast.map.QueryResultSizeExceededException;
+import com.hazelcast.map.impl.MapQueryResultSizeLimitHelper;
```

Comment:
Why don't you call this class:

MapQueryResultSizeLimitHelper ----> MapQueryResultSizeLimiter.

Every object helps out (or should be deleted :) ).